### ISSUE TITLE
Wrong links to the training data of the reader models

### ISSUE THREAD

**Issue by soheeyang** [AkariAsai/learning_to_retrieve_reasoning_paths#9]
In the README file for the reader (https://github.com/AkariAsai/learning_to_retrieve_reasoning_paths/blob/master/reader/README.md), [SQuAD reader train data] is linked to a private file (https://drive.google.com/open?id=186kFXd_g0pGJz12-E_sHvBVX6QrnyrzE), and [Natural Questions train data] is linked to a file that seems like the training data for SQuAD (https://drive.google.com/file/d/1aMTXIxYZCAC6sX5mZt6nytYxeKvjuigq/view).

**Comment by AkariAsai** [comment]
Thank you for reporting! We'll take a look and will fix it.

**Comment by jhyuklee** [comment]
Hi @AkariAsai, I'm also looking for NaturalQuestions training set for the reader model. The paper says that you used the original NQ training samples and I would like to know how you have pre-processed them. The original NQ samples are in HTML format (includes tables, lists, etc) and it's not easy to match their format to the format of articles from `build_db.py` with WikiExtractor. Thanks.

**Comment by AkariAsai** [comment]
@soheeyang I fixed the issue in the latest commit. Thank you for reporting the bug!

**Comment by jhyuklee** [comment]
It seems that the NQ link is still private.

**Comment by AkariAsai** [comment]
@jhyuklee I've updated the link. Please let me know if it does not work still now. 

Please note that we remove all of the tables and list elements from the NQ's original preprocessed HTML data, and only keep paragraph elements (elements surrounded by `<p>...</p>`).

**Comment by jhyuklee** [comment]
@AkariAsai Thanks for sharing! Could you also share the development set part? (used for NQ-open test set)

**Comment by AkariAsai** [comment]
Hmmm, I think we did not create the same data for Natural Questions dev ( = test data for the open-domain setting), as we do not use the gold docs for open-domain evaluation, and thus we cannot share the one for the original dev set... Sorry about that!

**Comment by jhyuklee** [comment]
Thanks! That helped a lot :)

**Comment by AkariAsai** [comment]
No problem. Wishing you all the best with the project! :)
